src/components/CalendarModal/CalendarModal.module.scss

```
@@ -6,22 +6,66 @@
   height: 100%;
   background: rgba(0, 0, 0, 0.6);
   text-align: center;
-  z-index: 1;
+  z-index: 10;
 }
 
 .modal-main {
   padding: 1rem;
   position: fixed;
   background: white;
-  width: 80%;
+  max-width: 500px;
```

```suggestion
  max-width: 32rem;
```
Nitpick: Let's change to `rem` units for better responsiveness (1rem = 16px)